gpkit/constraints/model.py

```diff
@@ -133,7 +134,7 @@ def _add_modelname_tovars(self, name, num):
         with SignomialsEnabled():  # since we're just substituting varkeys.
             self.subinplace(add_model_subs)
 
-    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-locals,too-many-branches,too-many-statements
```

Comment:
I'm never quite sure I fully understand pylint, but I think if you move this one line down it will disable within the `debug` method instead of for the whole file